README.md

```diff
@@ -161,7 +161,27 @@ Composer works great in combination with [Swarmer][swarmer] — another tool we'
 
 ### How to build
 
-Dependencies: you only need `docker` and `bash` installed on your machine.
+#### To build composer
+
+environmental variable ANDROID_HOME must be set.
```

Comment:
Capital `E`, wrap `ANDROID_HOME` with ` :)

Reply:
And afaik it's called environment variable, not environmental hehe

Reply:
:greenpeace: